### PR TITLE
Steps without Time attribute

### PR DESCRIPTION
**Pull request by Haoyuli2002**
Now I have deleted the Time attribute of the class Step and I allowed the user to create a step in the format like "s/cook d/1.6hours" so that a step with the description "cook" and an estimated Duration of 96 mins will be added to the RecipeStepLists. And also I created the corresponding Junit Tests for it, it seems to work well

### CODE REVIEW

File: src/main/java/essenmakanan/command/AddRecipeCommand.java [hunk 1]
```diff
@@ -95,16 +95,21 @@ public void addWithTitleStepsTags() {
     private static void addSteps(String step, RecipeStepList recipeStepList, Tag tag) {
         String[] allSteps = step.trim().split("s/");
         for (String eachStep : allSteps) {
-            if (eachStep.length()>0) {
+            if (eachStep.length()>0 && eachStep.contains("d/")) {
```
Comment (StanleyW00) [inline src/main/java/essenmakanan/command/AddRecipeCommand.java:98]: I think `eachStep.length()>0` should be `eachStep.length() > 0` for easier readability.

File: src/main/java/essenmakanan/parser/RecipeParser.java [hunk 2]
```diff
@@ -107,4 +107,26 @@ public static String parseFilterRecipeInput(String input) throws EssenFormatExce
...
+        if (time.contains("minutes") || time.contains("mins")) {
+            time = time.replace("minutes", "")
+                .replace("mins", "")
+                .trim();
+            return Integer.parseInt(time);
+        } else if (time.contains("hours") || time.contains("h") || time.contains("hour")) {
+            time = time.replace("hours", "")
+                .replace("h", "")
+                .replace("hour", "")
+                .trim();
```
Comment (StanleyW00) [inline src/main/java/essenmakanan/parser/RecipeParser.java:125]: I like that you considered more than one way of inputting the time for each step.

**Review by StanleyW00 — APPROVED** [review]
Overall, LGTM. Good job!

File: src/test/java/essenmakanan/parser/AddRecipeCommandTest.java [hunk 3]
```diff
@@ -51,6 +51,34 @@ public void addWithTitleStepsTags_validInput() {
         assertEquals(Tag.ACTUAL_COOKING, step5.getTag());
     }
 
+    @Test
+    public void addWithTitleStepsTags_validInput_duration() {
```
Comment (charkty) [inline src/test/java/essenmakanan/parser/AddRecipeCommandTest.java:55]: nice! you could consider adding invalid inputs as well

**Review by charkty — COMMENTED** [review]
LGTM but some changes to be made.

**Pull request merged**